### PR TITLE
AdaptaviveCards Adjusment

### PR DESCRIPTION
**Pull request by TheFab21**

### CODE REVIEW

File: galette/lib/Galette/IO/PdfMembersCards.php [hunk 1]
```diff
@@ -68,6 +68,14 @@ class PdfMembersCards extends Pdf
...
     protected float $hlogo;
     protected string $logofile;
+    protected int $nb_char;
+	protected int $avg_ratio;
+	protected float $ratio;
+	protected float $wphoto;
+    protected float $hphoto;
+	protected string $adh_nbr;
+	protected float $cell_he;
+	protected int $ban_max_he;
```
Comment (trasher) [inline galette/lib/Galette/IO/PdfMembersCards.php:78]: Those ones should be declared in `PdfMembersCardsAdaptative.php` file

**Review by trasher — CHANGES_REQUESTED** [review]
Please fix CS (running `vendor/bin/phpbcf` should do the trick.)

File: galette/lib/Galette/IO/PdfMembersCards.php [hunk 2]
```diff
@@ -68,6 +68,8 @@ class PdfMembersCards extends Pdf
     protected float $wlogo;
     protected float $hlogo;
     protected string $logofile;
+    protected int $nb_char;
```
Comment (trasher) [inline galette/lib/Galette/IO/PdfMembersCards.php:71]: Must be declared in `PdfMembersCardsAdaptative.php`

File: galette/lib/Galette/IO/PdfMembersCardsAdaptative.php [hunk 3]
```diff
@@ -36,6 +36,13 @@
  * @author Johan Cwiklinski <johan@x-tnd.be>
  * @author Fabrice Santoni <fabrice@santoni.ch>
  */
+protected int $avg_ratio;
+protected float $ratio;
+protected float $wphoto;
+protected float $hphoto;
+protected string $adh_nbr;
+protected float $cell_he;
+protected int $ban_max_he;
```
Comment (trasher) [inline galette/lib/Galette/IO/PdfMembersCardsAdaptative.php:45]: Incorrect. Those must be inside the class.

**Comment by trasher** [comment]
I did not made much tests, it seems better globally.

I continue to have override with due date:
image

Also, this date is very huge... But that's not a blocker, I'll merge this one.

**Comment by TheFab21** [comment]
Ok je vois le pb deadline ca me prendra 5 minutes à corriger 😉 quand c’est juste l’année ca va bien

 

De : Johan Cwiklinski ***@***.***> 
Envoyé : jeudi, 10 octobre 2024 22:08
À : galette/galette ***@***.***>
Cc : TheFab21 ***@***.***>; Author ***@***.***>
Objet : Re: [galette/galette] AdaptaviveCards Adjusment (PR #540)

 

I did not made much tests, it seems better globally.

I continue to have override with due date:
image.png (view on web) <https://github.com/user-attachments/assets/034c66e9-a5cd-4750-a71a-1f435c35f043> 

Also, this date is very huge... But that's not a blocker, I'll merge this one.

**Pull request merged**